client/src/components/signup/SignupFormField.js

```diff
@@ -0,0 +1,40 @@
+import styles from '../../styles/Signup-styles/SignupFormField.module.css'
+
+function SignupFormField({
+  formChoice,
+  iconSrc,
+  type,
+  placeholder,
+  name,
+  handleInput,
+  value,
+  error,
+}) {
+  return (
+    <div className={styles['SignupFormField__input-container']}>
+      <input
+        type={type}
+        aria-label={placeholder}
+        placeholder={placeholder}
+        name={name}
+        value={value}
+        onChange={(event) => handleInput(event)}
```

Comment:
Again, you can just add a pointer to the `handleInput` function without passing the event object like this. JavaScript is clever enough to pass it in for you.

Reply:
Nice one for catching that, didn't know JS does it for you automatically. Today I learnt. Fixed.